code/controllers/subsystem/blackbox.dm

```diff
@@ -8,7 +8,8 @@ SUBSYSTEM_DEF(blackbox)
 	var/list/feedback = list()	//list of datum/feedback_variable
 	var/triggertime = 0
 	var/sealed = FALSE	//time to stop tracking stats?
-	var/list/versions = list("time_dilation_current" = 2,
+	var/list/versions = list("admin_secrets_fun_used" = 2,
+							"time_dilation_current" = 3,
 							"science_techweb_unlock" = 2) //associative list of any feedback variables that have had their format changed since creation and their current version, remember to update this

```

Comment:
I'm wondering if the version would make more sense as an optional parameter to record_feeback...